lib/model.d.ts

```diff
@@ -448,10 +448,21 @@ export interface FindOptions {
    * Use sub queries (internal)
    */
   subQuery?: boolean;
+}
 
+export interface RejectOnEmpty {
```

Comment:
this interface seems pointless to ne

Reply:
This was done to avoid duplication of the property/documentation.